cpp/src/io/parquet/reader_impl.cu

```diff
@@ -1228,7 +1253,10 @@ rmm::device_buffer reader::impl::decompress_page_data(
                                stream.value()));
     }
   }
-  stream.synchronize();
+
+  decompress_check(inflate_out_view, stream, any_block_failure.device_ptr());
+  any_block_failure.device_to_host(stream, true);  // synchronizes stream
+  CUDF_EXPECTS(any_block_failure[0] == false, "Error during snappy decompression");
```

Comment:
"Error during decompression"

Reply:
Fixed, as well as the other couple cases where I had this error message